.github/workflows/check_tutorials.yml

```diff
@@ -29,8 +29,8 @@ jobs:
     # https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#specifying-a-python-version
     strategy:
       matrix:
-        # Only check the latest version of 
-        python-version:  ["3.12"]
+        # Only run check the latest supported version of Python
```

Comment:
"run check the latest..." doesn't make grammatical sense

Reply:
Updated.